spec/unit/mutant/env_spec.rb

```diff
@@ -45,18 +45,38 @@
     end
 
     before do
-      expect(selector).to receive(:call).with(mutation_subject).and_return(tests)
-      allow(Time).to receive(:now).and_return(Time.at(0))
+      expect(selector).to receive(:call)
+        .with(mutation_subject)
+        .and_return(tests)
+
+      allow(Time).to receive_messages(now: Time.at(0))
     end
 
     context 'when isolation does not raise error' do
       let(:test_result)  { instance_double(Mutant::Result::Test, passed: false) }
 
       before do
-        expect(isolation).to receive(:call).and_yield.and_return(test_result)
-        expect(mutation_subject).to receive(:prepare).and_return(mutation_subject).ordered
-        expect(context).to receive(:root).with(s(:nil)).and_return(wrapped_node).ordered
-        expect(Mutant::Loader::Eval).to receive(:call).with(wrapped_node, mutation_subject).and_return(nil).ordered
+        expect(isolation).to receive(:call)
+          .ordered
+          .and_yield
+
+        expect(mutation_subject).to receive(:prepare)
+          .ordered
+          .and_return(mutation_subject)
+
+        expect(context).to receive(:root)
+          .with(s(:nil))
+          .and_return(wrapped_node)
+
+        expect(Mutant::Loader::Eval).to receive(:call)
+          .ordered
+          .with(wrapped_node, mutation_subject)
+          .and_return(nil)
```

Comment:
This is just a leftover. I replaced it with returning the receiver to mirror what the real `Mutant::Loader::Eval` does.